openreview/arr/webfield/programChairsWebfield.js

```diff
@@ -186,14 +186,32 @@ return {
           });
           return replyToMap;
         }
+
+        const getAuthorRebuttals = (reply, note) => {
```

Comment:
I still don't understand why you are doing this?

Can we just iterate the array of replies and check the authors in readers/signatures? Why do you need to use while and update the queue object in the loop?